### PR TITLE
Fix spelling in collect_metrics() error

### LINKED ISSUE #770: `collect_metrics()` error misspells functions name
https://github.com/tidymodels/tune/blob/7155350225b29f05c3779d8a0408a7986e971919/R/collect.R#L424

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
I went a little over the brief, and converted all 4 of the errors to use cli (and fix the typo) while I was at it

to close #770

### CODE REVIEW

File: R/collect.R [hunk 1]
```diff
@@ -64,10 +64,10 @@
 #' However, this means that if a function was supplied to a
 #' [control function's][control_grid()] `extract` argument, tune can only
 #' execute that extraction on the one model that was fitted. As a result,
-#' in the `collect_extracts()` output, tune opts to associate the 
-#' extracted objects with the hyperparameter combination used to 
-#' fit that one model workflow, rather than the hyperparameter 
-#' combination of a submodel. In the output, this appears like
```
Comment (simonpcouch) [inline R/collect.R:70]: Are there `.Rd` changes associated with these deletions?

**Review by simonpcouch — APPROVED** [review]
Looks great, thanks!🙂

File: R/collect.R (hunk 1 above)
Comment (EmilHvitfeldt) [reply R/collect.R:70]: no :( i couldn't figure out how to get rid of it

Comment (simonpcouch) [reply R/collect.R:70]: all good!

**Pull request merged**